### PR TITLE
Viewenrollment migration

### PR DESCRIPTION
**Pull request by AnnaxGrace**
All of the courseAvailabilities updated, as well as editSessionView, SessionView, and enrollmentView updated to graphQL migration.

### CODE REVIEW

File: src/components/FeatureViews/Scheduler/EditSessionView.js [hunk 1]
```diff
@@ -1,359 +1,660 @@
...
+
+            <Grid item xs={12}>
+              <Typography variant="h5"> Instructor </Typography>
+              <SearchSelect
+                onChange={handleInstructorChange}
+                options={instructorList}
+                placeholder="Choose an Instructor"
+                value={sessionFields.instructor}
+              />
+              {location.state.allOrCurrent === "current" && (
```
Comment (dhuangg) [inline src/components/FeatureViews/Scheduler/EditSessionView.js:560]: make a wrapper component that helps make this more readable. For example:
```
<EditCurrentSessionField>
    <FormControl>
         <InputLabel> Is Instructor confirmed?</InputLabel>
    </FormControl>
</EditCurrentSessionField>
```
so the `<EditCurrentSessionField>` would look something like
```
const EditCurrentSessionField = ({children}) => {
    const location = useLocation();
    if(location.state.allOrCurrent)
          return children;
    else
           return;
}
```

File: src/components/FeatureViews/Scheduler/EditSessionView.js [hunk 2]
```diff
@@ -1,359 +1,660 @@
...
+                  >
+                    <MenuItem value>Yes, Instructor Confirmed.</MenuItem>
+                    <MenuItem value={false}>
+                      No, Instructor is NOT Confirmed.
+                    </MenuItem>
+                  </Select>
+                </FormControl>
+              )}
+            </Grid>
+            {location.state.allOrCurrent === "current" && (
```
Comment (dhuangg) [inline src/components/FeatureViews/Scheduler/EditSessionView.js:577]: use the `<EditCurrentSessionField/>` once you make it

File: src/components/FeatureViews/Scheduler/RETIRED_SessionView.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+import Paper from "@material-ui/core/Paper";
+import {GET} from "../../../actions/actionTypes";
+import DisplaySessionView from "./DisplaySessionView";
+import EditSessionView from "./EditSessionView";
+import {withRouter} from "react-router-dom";
+
+export const EDIT_ALL_SESSIONS = "all";
+export const EDIT_CURRENT_SESSION = "current";
+
+class RETIRED_SessionView extends Component {
```
Comment (dhuangg) [inline src/components/FeatureViews/Scheduler/RETIRED_SessionView.js:24]: delete this file

File: src/components/FeatureViews/Scheduler/EditSessionView.js [hunk 4]
```diff
@@ -1,359 +1,660 @@
...
+
+  const courseDurationOptions = [1, 1.5, 2, 0.5];
+
+  const course = data.session.course;
+
+  const session = data.session;
+
+  return (
+    <Grid container className="main-session-view">
+      <BackgroundPaper
```
Comment (dhuangg) [inline src/components/FeatureViews/Scheduler/EditSessionView.js:506]: remove `<BackgroundPaper/>`

File: src/components/FeatureViews/Scheduler/SessionView.js [hunk 5]
```diff
@@ -1,154 +1,389 @@
...
+                <Typography variant="body">
+                  No students enrolled yet.
+                </Typography>
+              )}
+            </Grid>
+          </Grid>
+          <Grid item xs={6}>
+            <Typography variant="h5">Day(s)</Typography>
+            <Typography>
+              {availabilityList.map(({ dayOfWeek }, index) => {
```
Comment (dhuangg) [inline src/components/FeatureViews/Scheduler/SessionView.js:281]: use Andrew's new component

**Review by dhuangg — CHANGES_REQUESTED** [review]
looking good so far! Let me know if you have any questions on my comments

File: src/components/FeatureViews/Scheduler/SessionView.js (hunk 5 above)
Comment (AnnaxGrace) [reply src/components/FeatureViews/Scheduler/SessionView.js:281]: So Andrew's component renders the time as T/TH, 10:00am - 11:00am. In this component you have a section for Day(s) and Time. Please see below. So for components that separate the days and time I used the functions within the component. Should I update the stylings to use Andrew's component? Or should I leave as is?
Screen Shot 2020-12-01 at 5 49 26 PM

Comment (AnnaxGrace) [reply src/components/FeatureViews/Scheduler/SessionView.js:281]: Note: this is an earlier image of this component, under days it would now say T/Th

Comment (dhuangg) [reply src/components/FeatureViews/Scheduler/SessionView.js:281]: hm. this is a good question actually. @katiebelleho what do you think?

Comment (AnnaxGrace) [reply src/components/FeatureViews/Scheduler/SessionView.js:281]: This was resolved in teams, with keeping the stylings as is.

File: src/components/FeatureViews/Scheduler/EditSessionView.js (hunk 4 above)
Comment (AnnaxGrace) [reply src/components/FeatureViews/Scheduler/EditSessionView.js:506]: Removed!

File: src/components/FeatureViews/Scheduler/RETIRED_SessionView.js (hunk 3 above)
Comment (AnnaxGrace) [reply src/components/FeatureViews/Scheduler/RETIRED_SessionView.js:24]: Deleted!

File: src/components/FeatureViews/Scheduler/EditSessionView.js (hunk 2 above)
Comment (AnnaxGrace) [reply src/components/FeatureViews/Scheduler/EditSessionView.js:577]: Component made and implemented!

File: src/components/FeatureViews/Scheduler/EditSessionView.js (hunk 1 above)
Comment (AnnaxGrace) [reply src/components/FeatureViews/Scheduler/EditSessionView.js:560]: Component made and implemented!

File: src/components/FeatureViews/Scheduler/EditSessionView.js [hunk 6]
```diff
@@ -1,359 +1,660 @@
...
+    const {
+      start_time,
+      room,
+      is_confirmed,
+      instructor,
+      category,
+      duration,
+      title,
+    } = sessionFields;
+    let newEndTime = calculateEndTime(duration, start_time);
```
[the remark this answers is not in the document]
Comment (AnnaxGrace) [reply src/components/FeatureViews/Scheduler/EditSessionView.js:445]: Updated to new name!

**Review by dhuangg — APPROVED** [review]
Awesome job Anna! Good to merge :)

**Pull request merged**